### PR TITLE
donut chart

### PR DESCRIPTION
**Pull request by SharonV33**
create atom component to render donut charts based on props

### CODE REVIEW

File: src/components/templates/Home.svelte [hunk 1]
```diff
@@ -1,18 +1,147 @@
...
+      'size': 39,
+      'name': 'Anders',
+      'color': '#88E8F2'
+    },
+    {
+      'size': 11,
+      'name': 'Anders',
+      'color': '#ac9fc2'
+    }
+  ]
```
Comment (veerleprins) [inline src/components/templates/Home.svelte]: Maybe it is better to put this 'test' data with comments in the DonutChart.svelte itself 😄 . This also prevents merge conflicts (I think)

File: src/components/atoms/DonutChart.svelte [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    //call the d3 function arc() to create arcs
+    let fn = arc()
+
+    //create a full circle
+    let angle = Math.PI * 2
+
+    let total = data.reduce((total, item) => total + item.size, 0)
+    let arcs
+    let buildDonut = () => {
```
Comment (veerleprins) [inline src/components/atoms/DonutChart.svelte]: I think this function is unnecessary. Maybe you can remove it 😄 ?

**Review by veerleprins — CHANGES_REQUESTED** [review]

**Review by veerleprins — APPROVED** [review]
Looks goot to me 👍🏼

**Review by johancvb — APPROVED** [review]
👍

**Pull request merged**